apps/condo/domains/common/utils/codegeneration/generate.hooks.ts

```diff
@@ -18,29 +18,30 @@ interface IHookConverters<GQL, GQLInput, UI, UIForm> {
 
 interface IHookResult<UI, UIForm, Q> {
     gql: any
-    useObject: (variables: Q, memoize?: boolean) => { obj: UI, loading: boolean, error?: ApolloError | string, refetch?: Refetch<Q> }
-    useObjects: (variables: Q, memoize?: boolean) => { objs: UI[], count: number | null, loading: boolean, error?: ApolloError | string, refetch?: Refetch<Q> }
+    useObject: (variables: Q) => { obj: UI, loading: boolean, error?: ApolloError | string, refetch?: Refetch<Q> }
+    useObjects: (variables: Q, queryOptions?: Record<string, unknown>) => { objs: UI[], count: number | null, loading: boolean, error?: ApolloError | string, refetch?: Refetch<Q> }
```

Comment:
you can import type from Apollo lib

Reply:
I replaced queryOptions with fetchPolicy (with apollo type FetchPolicy )